routers/router.go

```diff
@@ -84,4 +84,8 @@ func initAPI() {
 	beego.Router("/api/add-token", &controllers.ApiController{}, "POST:AddToken")
 	beego.Router("/api/delete-token", &controllers.ApiController{}, "POST:DeleteToken")
 	beego.Router("/api/login/oauth/access_token", &controllers.ApiController{}, "POST:GetOAuthToken")
+
+	beego.Router("/api/get-records", &controllers.ApiController{}, "GET:GetRecords")
+	beego.Router("/api/get-records-filter", &controllers.ApiController{}, "POST:GetRecordsByFilter")
 }
```

Comment:
I think only need to add one api, the filter conditions can be given in the url query parameter, like  `/api/get-records?ip=8.8.8.8&username=sh1luo&xxx=xxx...`

You only need to parse out the conditions within url.

Reply:
> I think only need to add one api, the filter conditions can be given in the url query parameter, like `/api/get-records?ip=8.8.8.8&username=sh1luo&xxx=xxx...`
> 
> You only need to parse out the conditions within url.

In other codes, direct access to all data is separate from filtering based on conditions. I think a consistent style should be maintained